examples/simple-game-server/main.go

```diff
@@ -787,8 +795,12 @@ func listContains(s *sdk.SDK, listName string, value string) (string, error) {
 	ok, err := s.Alpha().ListContains(listName, value)
 	if err != nil {
 		log.Printf("Error getting List %s contains value %s: %s", listName, value, err)
+		return strconv.FormatBool(ok), err
+	}	
+	if ok {
+		return "FOUND: " + strconv.FormatBool(ok) + "\n", nil
```

Comment:
I think this review comment got lost somewhere - but this result should be the following.

```go
func listContains(s *sdk.SDK, listName string, value string) (string, error) {
	log.Printf("Getting List %s contains value %s", listName, value)
	ok, err := s.Alpha().ListContains(listName, value)
	if err != nil {
		log.Printf("Error getting List %s contains value %s: %s", listName, value, err)
		return strconv.FormatBool(ok), err
	}	
  return "FOUND: " + strconv.FormatBool(ok) + "\n", nil
}
```

Not finding a value shouldn't be an error - it's just a search function.